CS342MemberViews/Sources/JenniferXu/JenniferXu.swift

```diff
@@ -0,0 +1,69 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+// Jennifer Xu Member View. Modeled partially after Paul Schmiedmayer's Member View.
+//
+// Cited:
+// GitHub Logo: GitHub Logos and Usage - https://github.com/logos
+// LinkedIn Logo: LinkedIn Brand Guidelines - https://brand.linkedin.com/downloads
+//
+//  Created by Jennifer Xu on 1/19/23.
+//
```

Comment:
@PSchmiedmayer @vishnuravi I'm not sure if I did the licensing correctly. Any advice is appreciated!